mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -802,36 +802,10 @@ fn dex_fee_rate(base: &str, rel: &str) -> MmNumber {
 pub fn dex_fee_amount(base: &str, rel: &str, trade_amount: &MmNumber, min_tx_amount: &MmNumber) -> DexFee {
     let rate = dex_fee_rate(base, rel);
     let fee = trade_amount * &rate;
-
     if &fee <= min_tx_amount {
         return DexFee::Standard(min_tx_amount.clone());
     }
 
-    if base == "KMD" {
-        // Drop the fee by 25%, which will be burned during the taker fee payment.
-        //
-        // This cut will be dropped before return if the final amount is less than
-        // the minimum transaction amount.
-
-        // Fee with 25% cut
-        let new_fee = &fee * &MmNumber::from("0.75");
-
-        let (fee, burn) = if &new_fee >= min_tx_amount {
-            // Use the max burn value, which is 25%.
-            let burn_amount = &fee - &new_fee;
-
-            (new_fee, burn_amount)
-        } else {
-            // Burn only the exceed amount because fee after 25% cut is less
-            // than `min_tx_amount`.
-            let burn_amount = &fee - min_tx_amount;
-
-            (min_tx_amount.clone(), burn_amount)
-        };
-
-        return DexFee::with_burn(fee, burn);
-    }
-
```

Comment:
could you clarify, why did you remove this?

Reply:
ah thanks for the catch...I actually didn't meant to